### PR TITLE
Improve select menu integration

### PR DESCRIPTION
**Pull request by josh**
See comments for explanations.

### CODE REVIEW

File: contentScript.js [hunk 1]
```diff
@@ -118,14 +118,14 @@
       itemList.appendChild(item);
       item.toolbar = toolbar;
       item.answer = answers[i].description;
-      item.addEventListener('click', insertAnswer);
+      $(item).on('selectmenu:selected', insertAnswer);
```
Comment (josh) [inline contentScript.js]: Can a Chrome extension see the `github.com`'s document `window.$`?

**Comment by josh** [comment]
Just did some blaming on the original select menu stuff. I originally wrote this all back around Feb 2012. It's lasted for a while.

**Comment by josh** [comment]
You also might want to actually test all this.

File: contentScript.js (hunk 1 above)
Comment (notwaldorf) [inline contentScript.js]: Nope :(
screen shot 2016-01-29 at 3 36 28 pm

And `item.addEventListener('selectmenu:selected', insertAnswer);` doesn't seeeeem to work

Comment (josh) [inline contentScript.js]: Damn. Are there restrictions on `CustomEvent` `dispatchEvent` between extension sandboxes?

Like if I change `$(item).trigger('selectmenu:selected')` in our code to `item.dispatchEvent(new CustomEvent('selectmenu:selected'))` would that help?

**Comment by holman** [comment]
i don't know i've merged joshcode blind before and it works 100% of the time

File: contentScript.js (hunk 1 above)
Comment (notwaldorf) [inline contentScript.js]: I don't thiiiiink so. A CustomEvent is a legit event; should work!

Comment (josh) [inline contentScript.js]: Alright, I'll just revert this part of the change for now. I think the other class name changes should be safe.

**Comment by josh** [comment]
Alright, I think these class changes should fix the focus issues. Also @mdo says hi.

**Comment by notwaldorf** [comment]
Awesome! Thanks friends! 😻

**Comment by notwaldorf** [comment]
@josh does the PRs page work differently than Issues? The focus is still kind of weird on PRs :(

**Comment by josh** [comment]
PRs and Issues should be pretty similar. I think there might be some other navigation items somewhere screwing with that activation. Though, I would think some of my changes would help.

**Pull request merged**